### PR TITLE
doc: update getting started

### PR DESCRIPTION
**Pull request by dbkinder**
Work-in-progress to update the getting started information.

### CODE REVIEW

**Comment by dbkinder** [comment]
@tomlenth @craigmateo Here's the word-to-rst conversion to get us started on an updated getting started guide.  (It needs some work, so let's talk)

File: getting-started/index.rst [hunk 1]
```diff
@@ -1,31 +1,126 @@
...
+* HuggingFace API Token
+* Prometheus and Grafana (optional, needed for evaluation purposes)
 
-      1. Supported platforms
-      #. Deployment
-      #. ChatQnA customization.
+1. **Determine where you will run your solution.**
 
-   #. Complete Deployment Options (links to Deployment section)
+   a. Basic deployment options suitable for proof of concept includes single architectures such as:
```
Comment (mkbhanda) [inline getting-started/index.rst:38]: Make this a more general statement.
Basic deployment options suitable for proof of concept includes a single node, which is either a virtual machine, physical server, which could be local or in a Cloud Service Provider (CSP) environment or an AI PC with adequate memory that has a single container engine such as Docker or Podman. (by nature of the node description .. single architecture?). We have tested thus far on Intel Xeons servers that are X86-based, Intel Gaudi2, and Nvidia H100.  (if itemizing, treat all equal)

File: getting-started/index.rst [hunk 2]
```diff
@@ -1,31 +1,126 @@
...
+
+      * An AI enable PC.
```
Comment (mkbhanda) [inline getting-started/index.rst:40]: s/enable/enabled

File: getting-started/index.rst [hunk 3]
```diff
@@ -1,31 +1,126 @@
...
+      * A server with Intel Xeon Processors.
+
+   b. Production ready environments include:
+
+      * Onprem or cloud based environments (including environments with NVIDIA GPUs).
```
Comment (mkbhanda) [inline getting-started/index.rst:45]: s/based environments /based cluster environments running Kubernetes as the orchestrator.

Production environments are typically multi-node to handle scale and resiliency.
We have only tested Kubernetes environments, the flavor of Kubernetes does not matter. Currently at version 1.30

File: getting-started/index.rst [hunk 4]
```diff
@@ -1,31 +1,126 @@
...
+==========
+
+* Explore additional :ref:`GenAI Examples <GenAIExamples>`
+* Build your own solution using :ref:`GenAI microservices <GenAIMicroservices>`
+* Get involved and contribute to the :ref:`OPEA open source community <OPEA_community>`
+
+Like What You See?
+******************
+
+Do you have ideas and skills to build out genAI components, microservices, and
```
Comment (mkbhanda) [inline getting-started/index.rst:69]: s/compoents, microservices/component microservices

File: getting-started/index.rst [hunk 5]
```diff
@@ -1,31 +1,126 @@
...
+solutions? Want to be part of these  early stages of this evolving technology?
```
Comment (mkbhanda) [inline getting-started/index.rst:70]: s/solutions? Want to be part of these early stages of this evolving technology?/
solutions, their evaluation, or support? Want to be part of this evolving technology in its early stages?

File: getting-started/index.rst [hunk 6]
```diff
@@ -1,31 +1,126 @@
...
+
+Get Involved
+============
+
+Register for our mailing list:
```
Comment (mkbhanda) [inline getting-started/index.rst:75]: s/Register for our mailing list:/Register to join our mailing list:

File: getting-started/index.rst [hunk 7]
```diff
@@ -1,31 +1,126 @@
...
+  * General: https://lists.lfaidata.foundation/g/OPEA-announce
+  * Technical Discussions: https://lists.lfaidata.foundation/g/OPEA-technical-discuss
+
+Subscribe to working group mailing lists
```
Comment (mkbhanda) [inline getting-started/index.rst:79]: s/Subscribe to working group mailing lists/Subscribe to one or more of the working group mailing lists based on your interest:

File: getting-started/index.rst [hunk 8]
```diff
@@ -1,31 +1,126 @@
...
+  * Security https://lists.lfaidata.foundation/g/OPEA-Security
+
+Visit the :ref:`OPEA_Community` documentation for Contribution Guidelines and
+step by step instructions.
+
+Attend any of our community events and hackathons found at our
+`Community Events <https://opea.dev/community-events>`_ page and register to
+stay updated.
+
+Want to Go Deeper?
```
Comment (mkbhanda) [inline getting-started/index.rst:93]: is this more to become a partner? With logo/quote etc?

**Review by mkbhanda — CHANGES_REQUESTED** [review]

**Review by tomlenth — COMMENTED** [review]
This all looks good.  The one thing we need to think about is a way to place the generic "Get Started" aimed at developers who will consume these projects and "Like what you see" section that is aimed at contributors.  I initially envisioned a two column approach, but there's too much material for that.  Tabs may not work since there are headings in the body content of the tabbed material.  The main thing is that we don't want to have the contribution section buried below the fold.  Perhaps we need two "buttony" options at the top of the page that vectors viewers into "Try AI Solution" and "Contribute AI Solutions".

**Review by yinghu5 — APPROVED** [review]
LGTM

**Review by mkbhanda — CHANGES_REQUESTED** [review]

File: getting-started/index.rst [hunk 9]
```diff
@@ -1,31 +1,139 @@
...
 
-   #. Complete Deployment Options (links to Deployment section)
+1. **Determine where you will run your solution.**
+
+   a. Basic deployment options suitable for proof of concept using a single
+      node:
+
+      * An AI enabled PC with adequate memory, using a single container engine
+        such as Docker or Podman.
+      * A virtual machine or physical server, which could be local or in a Cloud
```
[the remark this answers is not in the document]
Comment (dbkinder) [reply getting-started/index.rst:47]: Could you provide what wording you'd like to see here instead?

File: getting-started/index.rst [hunk 10]
```diff
@@ -1,31 +1,139 @@
...
+
+      * Onprem or cloud based cluster environments running Kubernetes as the
+        orchestrator
+
+      We have tested using Kubernetes environments.
+
+2. **Deploy a simple chatbot application using the OPEA ChatQnA example into your
+   environment.**
+
+3. **Customize the chatbot with your own data (RAG model)**.
```
[the remark this answers is not in the document]
Comment (dbkinder) [reply getting-started/index.rst:64]: Here too. Could you provide the wording you'd like to see.

**Comment by mkbhanda** [comment]
@dbkinder, I wonder if we can write the Getting Started as a story, reconciling with the different personas who may engage.

There are essentially four steps to getting started with OPEA, of which steps 3 and 4 are optional. 
1) Determine where you want to run your first GenAI application
2) Which of the OPEA samples you want to run
3) (optional) Whether and how you want to customize the OPEA sample application
4) (optional) Whether you want to evaluate the GenAI sample for accuracy, hallucinations, safety, bias etc. 

For the purposes of getting started, let us focus on a common use case: a ChatBot, which is covered by the ChatQnA sample. Further let us assume, as an end user you want to just create a proof of concept solution, no customizations of evaluation. 

Getting-Started ChatQnA Proof-of-Concept Simple
1) Determine where you want to run your first GenAI application - "Single Node".
   The single node could be an AI PC, a server, or a virtual machine, as long as it has adequate memory.  Single node deployments are most easily supported using a docker file approach.
2) Which of the OPEA samples you want to run - ChatQnA (ChatBot common use case). The simplest of single nodes that are widely available are X-86 architecture based. This translates to using docket-compose-chat_QnA_on_Intel_Xeon.  -- Takes the to the read me.


Getting-Started ChatQnA production
For scalability and resiliency people use a cluster of machines and launch and monitor their workloads on the cluster using an Orchestrator such as Kubernetes. Towards production readiness, run your application on a Kubernetes cluster. Further, for performance, one may want to use accelerators in one's cluster like Intel Gaudi, Nvidia GPUs etc. We currently have images for our samples that are specific to Intel CPU, called Xeons, and Intel Accelerator Gaudi. To keep things simple we assume you  have a Kubernetes cluster of Intel Xeons which is readily available on most Cloud Service Providers (CSPs) (EKS, AKS, GKE) or on-prem. Running large machine learning models on accelerators gives higher performance.
How to setup a Kubernetes cluster (on  EKS, AKS, GKE link or on-prem using KubeSpray/KubeAdm/..)
Provide link to ChatQnA helm chart link.

Getting-Started ChatQnA production and enterprise ready.
In a production environment with potentially multiple GenAI applications, there is opportunity to share common services and save on resources and thus cost. The OPEA GenAI Microservices Connector  provides, in addition to service sharing, additional features such as changing a model dynamically, defining GenAI pipelines with sequential, parallel and conditional steps.
<link to readme to GMC> followed by <Link to ChatQnA with GMC on Xeons>

OPEA solutions, homing in on the "Enterprise" in Open Platform for Enterprise AI, supports integration with enterprise functionalities such as Authentication, Authorization, Logging, Tracing, Observability, Guardrails and more. Some of these we provide through Service Mesh solutions such as Istio.  <link to Authn N&Z>, <link to GuardRails/Gateway>.


While we have not covered steps 3 and 4, Customization and Evaluation, we hint at what is possible and available. 
**Customization**. The ChatQnA GMC example illustrates how one may change the model-id used in the embedder, retriever and LLM components in the pipeline. One might also customize a pipeline by changing aspects like the vector database being used, the chunking algorithm, introducing different or additional guardrails, and even the model serving framework.  The GenAIComps sub-project hosts currently supported options in each GenAI pipeline component category.

**Evaluation** We have some initial benchmarking scripts to test for performance and accuracy in GenAIEval.

**Comment by dbkinder** [comment]
We're going to abandon this PR in favor of a new one in the works...

**Comment by dbkinder** [comment]
Closed and updated in #171 instead.

**Pull request closed without merging**